tests/vcf_test.py

```diff
@@ -43,6 +45,65 @@ def test_header(self, genome_reader):
             assert line
             assert line[0] == "#"
 
+    def test_header(self, blocks, genome_reader):
+        """
+        GIVEN an interable of blocks
+        """
+        vcfgenerator = VCFMaker(genome_reader, {})
+
+        """
+        GIVEN a VCF generator
+        """
+        # generate vcf lines and qc info
+        vcf_lines = [line for line in vcfgenerator.generate_lines(blocks)]
+
+        """
+        WHEN generating a header
+        """
+        lines = tuple(
+            (
+                str(i)
+                for i in vcfgenerator.generate_header("2022-03-25 9:20 AM", "GSAMD-24v3-0-EA_20034606_A2", "GRCh38")
+            )
+        )
+        """
+        THEN it should produce valid header output
+        """
+        header_data = {}
+        for line in lines:
+            print(line)
```

Comment:
🔧 stray `print()` that should be removed. We should use a `logger.debug()` if we want to keep track of it for failing tests instead (probably needs an `import logging` and `logger = logging.getLogger(__name__)` to match)